doc/using_mxnet.rst

```diff
@@ -433,71 +432,88 @@ MXNet on SageMaker has support for `Elastic Inference <https://docs.aws.amazon.c
 The SageMaker MXNet Model Server
 ================================
 
-The MXNet Endpoint you create with ``deploy`` runs a SageMaker MXNet model server. The model server loads the model that was saved by your training script and performs inference on the model in response to SageMaker InvokeEndpoint API calls.
+The MXNet endpoint you create with ``deploy`` runs a SageMaker MXNet model server.
+The model server loads the model provided and performs inference on the model in response to SageMaker ``InvokeEndpoint`` API calls.
 
-You can configure two components of the SageMaker MXNet model server: Model loading and model serving. Model loading is the process of deserializing your saved model back into an MXNet model. Serving is the process of translating InvokeEndpoint requests to inference calls on the loaded model.
+You can configure two components of the model server: model loading and model serving.
+Model loading is the process of deserializing your saved model back into an MXNet model.
+Serving is the process of translating ``InvokeEndpoint`` requests to inference calls on the loaded model.
 
-As with MXNet training, you configure the MXNet model server by defining functions in the Python source file you passed to the MXNet constructor.
+You configure the MXNet model server by defining functions in the Python source file you passed to the ``MXNet`` or ``MXNetModel`` constructor.
```

Comment:
[Very Minor] This sounds a bit off to me.
How about:
* Configure [...]
* You can configure [...]
* To configure [..] define...

Reply:
fixed